src/propodesolvers.jl

```diff
@@ -0,0 +1,34 @@
+abstract QuODESolvers <: QuPropagatorMethod
+
+type_to_method = {:QuODE45 => ODE.ode45, :QuODE78 => ODE.ode78, :QuODE23s => ODE.ode23s}
```

Comment:
On v0.4 there is warning for this line since the curly brace syntax is deprecated. You can try `@compat Dict{Any,Any}(a=>b, ...)` (or `Dict{Symbol,Function}`) instead. I would also add `const`.